lib/queries/Team.gql

```diff
@@ -0,0 +1,17 @@
+query team($org: String!, $team: String!) {
+  organization(login: $org) {
+    login
+    team(slug: $team) {
+      name,
+      description,
+      members(first: 100) {
```

Comment:
Does this limit how many team members we receive?

Reply:
@cPhost Yeah, I plan to fix that in a later PR

Reply:
Cool! then :)